tests/test_jobqueue.py

```diff
@@ -313,6 +317,27 @@ async def test_run_daily(self, job_queue):
         assert self.result == 1
         scheduled_time = job_queue.jobs()[0].next_t.timestamp()
         assert scheduled_time == pytest.approx(expected_reschedule_time)
+        assert len(recwarn) == 1
+        assert str(recwarn[0].message) == expected_warning
+
+    async def test_run_daily_days_of_week(self, job_queue, recwarn):
```

Comment:
```suggestion
    @pytest.mark.parametrize('weekday', (0,1,2,3,4,5,6))
    async def test_run_daily_days_of_week(self, job_queue, recwarn, weekday):
```

With this the test will automatically be run for each of the six weekdays so that we can check that the behavior works as expected for all of them :) you can use the `weekday` variable in the test, e.g. as `job_queue.run_daily(…, day=[weekday])`

Reply:
I wanted to do this inside the test, but I just cant get a way to get the next item in the jobs list, thanks for this!

also, is there a better way to handle the weekday of the same day? I ended up with this as the offset
```python

        offset = (weekday + 6 - now.weekday()) % 7
        offset = offset if offset > 0 else 7
```

Reply:
I think that's fine. one can probably drop the second line (weekday + day >= 6, now.weekday() <= 6, so the difference will never be negative), but I'm also okay with keeping it. it's just tests 😄